user_journey/views/cohort_analysis/search_clients_last_seen.view.lkml

```diff
@@ -1,4 +1,4 @@
-view: clients_last_seen {
+view: search_clients_last_seen {
```

Comment:
Extending from `clients_last_seen` requires renaming this view which is also called `clients_last_seen`

Reply:
Got it, forgot this was `search_clients_last_seen`. Then it doesn't make sense to extend from it (different table)

Reply:
I do think it makes sense to extend from the `clients_last_seen` in looker-hub and rename the custom view here to `search_clients_last_seen`, then the naming would also be correct and cause less confusion.